packages/flutter/test/material/reorderable_list_test.dart

```diff
@@ -541,6 +541,99 @@ void main() {
           ));
           handle.dispose();
         });
+
+        testWidgets('Default scroll physics for macOS/iOS', (WidgetTester tester) async {
```

Comment:
I'm not exactly sure what the best way to test this is, since the scroll physics is passed all the way down to the `Scrollable` widget. It gets the `ScrollConfiguration` from its parent, and the problem with this test is that we're polling directly from the `ScrollConfiguration` instead of seeing whether or not the ReorderableListView is actively utilizing this ScrollConfiguration. 

It also appears that when the physics property is passed in, it's [applied to the platform's default scroll configuration](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/widgets/scrollable.dart#L364), so we'll need to capture that particular behavior.

Upon doing some investigation, I noticed that [`scrollable_test.dart`](https://github.com/flutter/flutter/blob/master/packages/flutter/test/widgets/scrollable_test.dart) contains a series of fling/drag tests for platform-specific behavior. I wonder if we can somehow use those as the basis for the tests here?

@xster, could you provide some pointers on how we should test this behavior, since you wrote the scrollable tests? I know you're on leave right now, so there's no rush in getting back to this ASAP -- I just wanted to put this on your radar if it's easy to provide a quick suggestion here.

Reply:
Since Xiao is on leave, maybe @goderbauer could provide a suggestion here? I don't want to keep you waiting for too long, but I'm still not sure how we should approach this test.

Reply:
This test isn't really testing that ReorderableListView has the default scroll physics. It's just testing that `ScrollBehavior.getScrollPhysics` returns the right thing, and that should be tested elsewhere.

One thing you could do: pass in a ScrollController to your ReorderableListView, which gives you access to `position` and from there you can access the current `ScrollPhysics`. Then you can assert that they are what you expect.

Another way to test is to build a ReorderableListView without any physics. Perform some drag gestures on it to assert that those behave as expected with the default physics. Then rebuild the ReorderableListView with some custom physics and re-do the drag gestures again expecting a different result now (based on the custom physics).